examples/account/account_create_transaction_without_alias.py

```
@@ -1,36 +1,52 @@
 
 Usage:
-- uv run python examples/account/account_create_transaction_without_alias.py
-- python examples/account/account_create_transaction_without_alias.py
+    uv run examples/account/account_create_transaction_without_alias.py
+    python examples/account/account_create_transaction_without_alias.py
 """
 
-from typing import Tuple
+import os
 import sys
+from typing import Tuple
```

Ruff (0.14.10)

`typing.Tuple` is deprecated, use `tuple` instead

(UP035)